packages/lab/src/navigation-item/ConditionalWrapper.tsx

```diff
@@ -25,7 +25,7 @@ export const ConditionalWrapper = ({
     onExpand?.(event);
   };
 
-  return parent ? (
+  return parent || href === undefined ? (
     <Button
```

Comment:
I think we are better off refactoring this to not use Salt's Button directly. Dependencies like this cause issues with CSS specificity and make overrides confusing to consumers.